src/test/integration/users/User_M2.t.sol

```diff
@@ -2,28 +2,11 @@
 pragma solidity ^0.8.27;
 
 import "forge-std/Test.sol";
-
-import "src/test/integration/deprecatedInterfaces/mainnet/IEigenPod.sol";
-import "src/test/integration/deprecatedInterfaces/mainnet/IEigenPodManager.sol";
-import "src/test/integration/deprecatedInterfaces/mainnet/IStrategyManager.sol";
-import "src/test/integration/deprecatedInterfaces/mainnet/IDelegationManager.sol";
-
+import "src/test/integration/deprecated/mainnet/IDelegationManager.sol";
```

Comment:
I think we _can_ delete this, but it might require just a little bit more work because it's still used (though we don't need to be using it).

Let's mark this work for a followup PR.